index.html

```diff
@@ -206,40 +212,19 @@ <h2>
         <pre class="nohighlight">
 web-did = "did:web:" domain-name
 web-did = "did:web:" domain-name * (":" path)
+web-did = "did:web:" domain-name * (":" path) * (":" filename)
         </pre>
 
         <pre class="example nohighlight" title="Example Web Method DIDs">
 did:web:w3c-ccg.github.io
-
 did:web:w3c-ccg.github.io:user:alice
-
 did:web:example.com%3A3000
+did:web:example.com%3A3000:bob%3Awork
+did:web:example.com:christine:111.json
+did:web:example.com:christine:222.didjson
```

Comment:
hmm, I feel like this one should be removed... since `application/did+json` is `application/json`....

The current method only supports JSON, these extensions should now appear in the identifier.

Reply:
If we want to add this capability, we should do so in a separate PR....